### ISSUE TITLE
Automatically remove/delete a specific entry from the cache

### ISSUE THREAD

**Issue by fopson** [MarkBaggett/apiify#2]
Hi,
Any ways to trigger the removal/expiry/deletion of a specific entry from the cache regardless of the cache size or the cache expiry date?

I am testing using Apiify to cache dynamic information and have been looking for a way to automatically remove the entry from the cache.

Let's say I am caching Mac address - IP address - Gateway based on DHCP logs and would like to delete the cached entry when the IP address is assigned to a different mac-address. Something like: `http://{apiify_svr_ip}:{port}/delete/{key}`?  To force Apiify to go get a more up-to-date value next time `http://{apiify_svr_ip}:{port}/{key}` is called.

Thanks

**Comment by MarkBaggett** [comment]
I added a /delete/key option for you.   Check it out and let me know how it works.

**Comment by fopson** [comment]
Hi @MarkBaggett, I gave it a try and it works like a charm, thank you!

Taking hints from the updates in the apiify class, I have tried:
 `/delete/{cached_key}` and it return "Deleted entry {cached_key}" and I checked the cache (`/cache`) and it was gone.
 `/delete/{same_key}` returned "Entry not found in cache.", as expected.
then `/{key}` created a new cached entry with an up-to-date value and new timestamps in the cache.

Thanks again for the quick response as always!
